jme3-plugins/src/xml/java/com/jme3/export/xml/XMLUtils.java

```diff
@@ -0,0 +1,109 @@
+ * met:
+ *
+ * * Redistributions of source code must retain the above copyright
+ *   notice, this list of conditions and the following disclaimer.
+ *
+ * * Redistributions in binary form must reproduce the above copyright
+ *   notice, this list of conditions and the following disclaimer in the
+ *   documentation and/or other materials provided with the distribution.
+ *
+ * * Neither the name of 'jMonkeyEngine' nor the names of its contributors
+ *   may be used to endorse or promote products derived from this software
+ *   without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED
+ * TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+ * PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+ * CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+ * EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+ * PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+ * PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+ * LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ * NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package com.jme3.export.xml;
+
+import com.jme3.export.FormatVersion;
+import org.w3c.dom.Element;
+
+/**
+ * Utilities for reading and writing XML files.
+ * 
+ * @author codex
+ */
+public class XMLUtils {
+    
+    /**
+     * Prefix for every jme xml attribute for format versions 3 and up.
+     * <p>
+     * This prefix should be appended at the beginning of every xml
+     * attribute name. For format versions 3 and up, every name to
+     * access an attribute must append this prefix first.
+     */
+    public static final String PREFIX = "jme-";
+    
+    /**
+     * Sets the attribute in the element under the name.
+     * <p>
+     * Automatically appends {@link #PREFIX} to the beginning of the name
+     * before assigning the attribute to the element.
+     * 
+     * @param element element to set the attribute in
+     * @param name name of the attribute (without prefix)
+     * @param attribute attribute to save
+     */
+    public static void setAttribute(Element element, String name, String attribute) {
+        element.setAttribute(PREFIX+name, attribute);
+    }
+    
+    /**
+     * Fetches the named attribute from the element.
+     * <p>
+     * Automatically appends {@link #PREFIX} to the beginning
+     * of the name before looking up the attribute for format versions 3 and up.
+     * 
+     * @param element XML element to get the attribute from
+     * @param name name of the attribute (without prefix)
+     * @return named attribute
+     */
+    public static String getAttribute(Element element, String name) {
+        if (FormatVersion.VERSION >= 3) {
```

Comment:
To support old files, the check should be against `importer.formatVersion`  that is the version of the imported file, and not `FormatVersion.VERSION` that is the current supported version.